testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayViewModel.kt

```diff
@@ -63,6 +63,31 @@ class MiniAppDisplayViewModel constructor(
         }
     }
 
+    fun obtainMiniAppDisplayUrl(
+        context: Context,
+        appUrl: String,
+        miniAppMessageBridge: MiniAppMessageBridge,
+        miniAppNavigator: MiniAppNavigator
+    ) = viewModelScope.launch(Dispatchers.IO) {
+        try {
+            _isLoading.postValue(true)
+            miniAppDisplay = miniapp.createWithUrl(appUrl, miniAppMessageBridge)
+            hostLifeCycle?.addObserver(miniAppDisplay)
+            _miniAppView.postValue(miniAppDisplay.getMiniAppView(context))
+        } catch (e: MiniAppSdkException) {
+            e.printStackTrace()
+            when (e) {
+                is MiniAppHasNoPublishedVersionException ->
```

Comment:
Do we need to catch this exception? Because `MiniApp.createWithUrl` doesn't throw this one.

Reply:
Oh I missed that one.